private/buf/cmd/buf/buf.go

```diff
@@ -175,6 +176,13 @@ func NewRootCommand(name string) *appcmd.Command {
 					modlsbreakingrules.NewCommand("ls-breaking-rules", builder),
 				},
 			},
+			{
+				Use:   "plugin",
+				Short: "Work with plugins",
```

Comment:
Should we specify check/WASM plugins here, since we are not supporting the Docker image/generation plugins yet?